package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "template-config",
-  "version": "4.12.6",
+  "version": "4.13.6",
```

Comment:
It had been 4.13.0, then I made some changes and bumped the patch a few times. I used `commit --amend` so the bumps are not shown here. I'll change it back to 4.13.0 if you'd like